### PR TITLE
4231 Store HTMLs from Case Query Crawl

### PR DESCRIPTION
**Pull request by albertisfu**
This PR adds support for storing the HTML files from the Case Query Crawl as PacerHtmlFiles instances.

HTML files are stored for both dockets retrieved by the probe daemon and cases retrieved by the sweep signal.

Fixes: #4231

### CODE REVIEW

**Review by mlissner — APPROVED** [review]
Seems fine to me. Assigning to @ERosendo so somebody can look more closely. :)

File: cl/corpus_importer/tasks.py [hunk 1]
```diff
@@ -1158,12 +1158,12 @@ def filter_docket_by_tags(
...
 def query_case_query_report(
     court_id: str, pacer_case_id: int
-) -> dict[str, Any]:
+) -> tuple[dict[str, Any], str]:
     """Query the iquery page for a given PACER case ID.
 
     :param court_id: A CL court ID where we'll look things up.
     :param pacer_case_id: The Pacer Case ID to lookup.
-    :return: The report.data.
+    :return: A twp tuple, the report data and the report HTML text.
```
Comment (ERosendo) [inline cl/corpus_importer/tasks.py:1166]:
```suggestion
    :return: A tuple containing a dictionary representing the parsed report data and the raw HTML content of the report
```

**Review by ERosendo — APPROVED** [review]
LGTM 👍

**Comment by mlissner** [comment]
I committed your typo suggestion and set this for auto-merge. Thank you!

**Pull request merged**